docs/topics/api/discovery.rst

```diff
@@ -19,6 +19,6 @@ Firefox (about:addons).
 
     :>json int count: The number of results for this query.
     :>json array results: The array containing the results for this query.
-    :>json string results[].heading: The heading for this item. May contain the following sub-strings, that clients need to use to format the string as they desire: ``{start_sub_heading}``, ``{end_sub_heading}`` and ``{addon_name}``.
+    :>json string results[].heading: The heading for this item. May contain some HTML tags.
```

Comment:
"some" feels not informative here. The only fact that matters is that it can contain HTML tags right?